### DISCUSSION TITLE
Welcome to Androapps Discussions #Hacktoberfest2021

### DISCUSSION THREAD

**Discussion by aadityamp01** [aadityamp01/Androapps#38] [Announcements]
## 👋 Hey folks!!
  We’re using this Discussions as a place to connect with other members of this inclusive community. We hope that you:
  * Ask questions about open source and quality contributions.
  * Share ideas.
  * Engage with other community members and participate by helping them.
  * Welcome others and are open-minded.

**Comment by AnuragThePathak** [comment]
Hello, if I reformat/optimize code in an existing project can I upgrade Kotlin and Gradle versions along with that?

**Comment by aadityamp01** [comment]
Hey @AnuragThePathak Older versions are working fine, if you observe any issue/bug create new issue with name and description plus attach images of it.
Also these simple apps are for beginners so we require java/ kotlin both, no need to migrate java to kotlin.
If you're experience developer please consider developing new app.

**Comment by AnuragThePathak** [comment]
Ok I got it.
